scripts/build_non_firebase_sdks.sh

```diff
@@ -30,7 +30,9 @@ do
   if [ ${sdk} == "FirebaseFirestoreSwift" ]; then
     echo "{\"name\":\"FirebaseFirestoreSwift\", \"version\" : \"> 8.2-beta\"}" >>  "${ZIP_POD_JSON}"
```

Comment:
unrelated to the RC quickstart work, but should this bump to 8.12 as well?
```suggestion
    echo "{\"name\":\"FirebaseFirestoreSwift\", \"version\" : \"> 8.12-beta\"}" >>  "${ZIP_POD_JSON}"
```

Reply:
Gotcha, thanks for the context!